pyenvbuilder/main.py

```diff
@@ -0,0 +1,51 @@
+'''
+pyenvbuilder's commands
+'''
+import argparse
+import logging
+from pyenvbuilder import __version__
+
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
We should go with lowercase `logger` as it is not a constant.

Reply:
got it